src/software/simulation/er_force_simulator_robot.cpp

```diff
@@ -29,21 +29,49 @@ void ErForceSimulatorRobot::setRobotState(const RobotState& robot_state)
 std::unique_ptr<SSLSimulationProto::RobotCommand> ErForceSimulatorRobot::getRobotCommand()
 {
     auto move_command = createRobotMoveCommand(
-        wheel_speed_front_right, wheel_speed_front_left, wheel_speed_back_left,
-        wheel_speed_back_right, robot_constants.front_wheel_angle_deg,
+        *direct_control, robot_constants.front_wheel_angle_deg,
         robot_constants.back_wheel_angle_deg, wheel_constants.wheel_radius_meters);
 
-    if (isAutochipEnabled())
+    // If there are robots on the field that don't have a primitive, the wheel control
+    // will be empty. In this case we return a blank RobotMoveCommand so that robots don't
+    // move.
+    if (direct_control->wheel_control_case() ==
+        TbotsProto::DirectControlPrimitive::WHEEL_CONTROL_NOT_SET)
     {
-        chip(autochip_distance_m.value());
+        move_command = std::make_unique<SSLSimulationProto::RobotMoveCommand>();
     }
```

Comment:
createRobotMoveCommand already handles this case

Reply:
true, fixed